tools/c7n_gcp/c7n_gcp/filters/__init__.py

```diff
@@ -4,4 +4,5 @@
 from .metrics import GCPMetricsFilter
 from .sccfindings import SecurityComandCenterFindingsFilter
 from .iampolicy import IamPolicyFilter
-from .alerts import AlertsFilter
\ No newline at end of file
+from .alerts import AlertsFilter
+from .accessapproval import AccessApprovalFilter
```

Comment:
resource specific filters go in the module that defined the resource, the filters package is typically for filters that can be used across multiple  resource types.

Reply:
Thx, Moved it to appropriate gcp resource package.